code-challenges.test.js

```diff
@@ -69,3 +150,25 @@ var hand3 = [5, 5, 5, 5, 4]
 
 
 // b) Create the function that makes the test pass.
+
+//declare a function that takes in an array
+//loops through arrays
+//write conditional statement
+//return whether the outcome is true or false.
+
+const fullHouse = (array) => {
+    let trueFalse = []
+    for(i=0; i<array.length; i++){
+        if(array === [5, 5, 5, 3, 3]){
```

Comment:
you're calling on specific values here , what if I had aces and jacks? would your code be able to handle this? keep your functions dynamic and you'll have an easier time keeping track of what you're doing. 

Also, your pseudo code is a good summary of what you have done but it should be the path you are gonna follow. for every line of code you should have a line of pseudo code... more or less.